functions/idvtoauth0/main.py

```diff
@@ -32,6 +32,23 @@ def find_user(user_id):
         return None
 
 
+def _denullify_empty_values(data):
+    """
+    Opposite of https://github.com/akatsoulas/cis/blob/8c8da24b2c215d02f5e14dec7a94da6b1792c8c9/cis/publisher.py#L80
+    Remove `NULL` and replace it back by empty string
+    """
+    new = {}
+    for k in data.keys():
+        v = data[k]
+        if isinstance(v, dict):
+            v = _denullify_empty_values(v)
+        if v == 'NULL':
```

Comment:
When we are adding `NULL`, we do so for both empty strings and `None` values. I do not mind using empty strings for the reverse, I just want to point this out

Reply:
I don't believe `None` is a valid JSON value (`"None"` is, but it's understood as a literally string not as `None`) while empty string is a valid value and is understood as empty.
The alternative is `null` (no capital) in JSON, though again, I believe this may break end-to-end signatures in the future, as code or libraries sometimes understand an attribute with value `null` as a non-existent attribute and does not pass the key (ie `{ "blah": null` becomes `{}`)